erpnext/accounts/doctype/financial_report_template/financial_report_engine.py

```
@@ -0,0 +1,1805 @@
+		api_path = row.calculation_formula
+		# TODO
+
+		try:
+			values = frappe.call(api_path, filters=self.context.filters, periods=self.period_list, row=row)
+
+			if row.reverse_sign:
+				values = [-1 * v for v in values]
+
+			# TODO: add support for server script
+			# use form_dict to pass input in server script
+		except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)